### PR TITLE
addhub project with first story, project mgmt tool and UI details

### PR DESCRIPTION
**Pull request by pangjac**
create a link to project folder with addhub.md

### CODE REVIEW

File: projects/addhub.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+### Team
+* Jacqueline Pang https://github.com/Cornell-CS5356-Fall2015/cs5356/blob/master/people/jacqueline-pang.md
+* Sasinda Premarathna https://github.com/Cornell-CS5356-Fall2015/cs5356/blob/master/people/sasinda-rukshan.md
+* Yilin Xu https://github.com/Cornell-CS5356-Fall2015/cs5356/blob/master/people/yilin-xu.md
+
+### First Story
+* https://github.com/addhub/prototype/issues/1
+- As a person I want to go to addhub site and add an advertisement to sell something I want to sell.
```
Comment (dblock) [inline projects/addhub.md]: You don't need to repeat a story here. However the story is more like the "whole thing". A story should be very very basic and something you can implement in a week.

File: README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Projects
 See [Project Logistics](projects/logistics.md) for important dates and deliverables.
 
 #### Class Project Ideas
-
```
Comment (dblock) [inline README.md:41]: Eating our lines? ;)

**Comment by dblock** [comment]
Update the page wrt the first story please (remove it).

**Comment by pangjac** [comment]
handle the issues. Address changed.

**Comment by dblock** [comment]
Very nice.

**Pull request merged**